lib/elixir/test/elixir/enum_test.exs

```diff
@@ -428,6 +428,11 @@ defmodule EnumTest do
     assert Enum.sum([1]) == 1
     assert Enum.sum([1, 2, 3]) == 6
     assert Enum.sum([1.1, 2.2, 3.3]) == 6.6
+    assert Enum.sum([-3, -2, -1, 0, 1, 2, 3]) == 0
```

Comment:
My personal opinion is probably yes, but maybe it's ok since this is a small PR. Let's have @lexmag or @josevalim decide :)